lib/main.dart

```diff
@@ -34,7 +34,7 @@ class _MyAppState extends State<MyApp> {
       ),
       routes: {
         ""
-        "/": (context) => HomePage(),
+        "/": (context) => LoginPage(),
```

Comment:
Please import `LoginPage()` at the top of the file with `import 'package:growpal_hackathon/pages/LoginPage.dart';